src/pages/Staking/Staking.tsx

```diff
@@ -1,11 +1,14 @@
-import { BigNumber, ethers } from "ethers"
+import { BigNumber, ethers, utils } from "ethers"
 import React from "react"
 import { useDebounce } from "use-debounce"
 import AllowanceGate from "../../components/AllowanceGate/AllowanceGate"
 import { Box } from "../../components/Box"
 import { Button } from "../../components/Button/Button"
 import ConnectGate from "../../components/ConnectGate/ConnectGate"
-import useAmountState from "../../hooks/useAmountState"
```

Comment:
Are we not using this hook anymore?

Reply:
I have used the Input component that you've created (which uses this hook to store the amount), and in Staking I just store the final value that I get on the onChange event, similar to Fundraise page.

Reply:
ahhh of course, my brain isn't working properly. haha.